app/models/manageiq/providers/google/container_manager.rb

```diff
@@ -0,0 +1,40 @@
+ManageIQ::Providers::Kubernetes::ContainerManager.include(ActsAsStiLeafClass)
+
+class ManageIQ::Providers::Google::ContainerManager < ManageIQ::Providers::Kubernetes::ContainerManager
+  def self.ems_type
+    @ems_type ||= "gke".freeze
+  end
+
+  def self.description
+    @description ||= "Google Kubernetes Engine".freeze
+  end
+
+  def self.display_name(number = 1)
+    n_('Container Provider (Google)', 'Container Providers (Google)', number)
+  end
+
+  def self.default_port
+    443
+  end
+
+  def self.kubernetes_auth_options(options)
+    auth_options = {}
+    auth_options[:bearer_token] = google_access_token(options[:bearer])
+    auth_options
+  end
+
+  def self.google_access_token(google_json_key)
+    require "googleauth"
+    credentials = ::Google::Auth::ServiceAccountCredentials.make_creds(
```

Comment:
NOTE: kubeclient does have a [`Kubeclient::GoogleApplicationDefaultCredentials`](https://github.com/abonas/kubeclient/blob/e03dd5508f01bd632cee21e788318d790eabe187/lib/kubeclient/google_application_default_credentials.rb) credential type, but it uses [`Google::Auth.get_application_default(scopes)`](https://github.com/abonas/kubeclient/blob/e03dd5508f01bd632cee21e788318d790eabe187/lib/kubeclient/google_application_default_credentials.rb#L5) which only pulls creds from environment variables and known paths.  This way allows us to use the json service account from the database easier.